### DISCUSSION TITLE
V0.0.2 Released Today!

### DISCUSSION THREAD

**Discussion by FrankLeeeee** [hpcaitech/ColossalAI#230] [Announcements]
# Change Log

## Added

- Unifed distributed layers
- MoE support
- DevOps tools such as github action, code review automation, etc.
- New project official website

## Changes

- refactored the APIs for usability, flexibility and modularity
- adapted PyTorch AMP for tensor parallel
- refactored utilities for tensor parallel and pipeline parallel
- Separated benchmarks and examples as independent repositories
- Updated pipeline parallelism to support non-interleaved and interleaved versions
- refactored installation scripts for convenience

## Fixed

- zero level 3 runtime error
- incorrect calculation in gradient clipping

<hr /><em>This discussion was created from the release <a href='https://github.com/hpcaitech/ColossalAI/releases/tag/v0.0.2'>V0.0.2 Released Today!</a>.</em>

**Comment by Nliver** [comment]
**Congratulations**🎉

**Comment by binmakeswell** [comment]
Congratulations~🎉

**Comment by xxw11** [comment]
Congratulations🎉
